components/radio/stories/radio.stories.js

```diff
@@ -86,13 +92,14 @@ export default {
 	parameters: {
 		actions: {
 			handles: ["click input[type=\"radio\"]"],
-
 		},
 		componentVersion: version,
 	},
 };
 
-export const Default = RadioGroup.bind({});
+export const Default = (args, context) => context.viewMode === "docs"
```

Comment:
I had it as a separate "Standard" story previously like your example, but changed to how it is now in order to keep this as the story named "Default" (while also having the "Default" in the sidebar which has the single radio instead of the two radios). A previous comment mentioned the use of "Standard" naming as a synonym for "Default". And this seemed like the simplest way to use the same story name. @castastrophe do you have a preference here?

Reply:
Could we just change the display name of `Standard`? `Standard.storyName = "Default"`?

Reply:
I wasn't sure from the Storybook docs whether storyName should be unique; there was some conflicting info, but I think it can be. I'll try putting it back to that.